### PR TITLE
{bio}[foss/2016b] CNVnator 0.3.3

### PR DESCRIPTION
**Pull request by robqiao**
CNVnator easyconfig added

### CODE REVIEW

File: easybuild/easyconfigs/c/CNVnator/CNVnator-0.3.3-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = ['v%(version)s/%(name)s_v%(version)s.zip']
+source_urls = ['https://github.com/abyzovlab/CNVnator/releases/download']
+
+dependencies = [
+    ('Perl', '5.24.0'),
+]
+
+skipsteps = ['build']
```
Comment (boegel) [inline easybuild/easyconfigs/c/CNVnator/CNVnator-0.3.3-foss-2016b.eb]: hmm, it's kind of strange to use `MakeCp` together with skipping the `build` step...

Maybe you want `BinariesTarball` instead? Cfr. http://easybuild.readthedocs.io/en/latest/version-specific/generic_easyblocks.html#binariestarball

Comment (robqiao) [inline easybuild/easyconfigs/c/CNVnator/CNVnator-0.3.3-foss-2016b.eb]: @boegel Hmm, there are many other files inside the .zip like src, license etc. a perl script is really what is needed here.

Comment (boegel) [inline easybuild/easyconfigs/c/CNVnator/CNVnator-0.3.3-foss-2016b.eb]: In that case, I guess `CmdCp` is a better match, see http://easybuild.readthedocs.io/en/latest/version-specific/generic_easyblocks.html#cmdcp .

You'll still need `skipsteps = ['build']`, but it makes more sense to use `CmdCp` than `MakeCp` imho.

Comment (robqiao) [inline easybuild/easyconfigs/c/CNVnator/CNVnator-0.3.3-foss-2016b.eb]: @boegel sounds good to me. Thx

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2689.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/83961e267200c51f8427a01a2ae8843c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2135.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9da66b82435a26441a1e547f03bd8f37 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @robqiao!

**Pull request merged**